sideboard/config.py

```diff
@@ -38,36 +99,64 @@ def get_dirnames(pyname, plugin):
     package under the configured "plugins_dir" directory.
 
     These two directories are also automatically inserted into plugin config
-    files as "root" and "module_root" and are available for interpolation.  For
-    example, a plugin could have a line in their config file like
+    files as "root" and "module_root" and are available for interpolation. For
+    example, a plugin could have a line in their config file like::
+
         template_dir = "%(module_root)s/templates"
+
+
     and that would be interpolated to the correct absolute path.
 
-    This function takes two parameters:
-    pyname: the __file__ of the module needing to know these directories
-    plugin: boolean indicating whether this is a plugin or Sideboard itself
+    Parameters:
+        requesting_file_path (str): The __file__ of the module requesting the
+            "root" and "module_root" directories.
+        is_plugin (bool): Indicates whether a plugin is making the request or
+            Sideboard itself is making the request.
     """
-    module_dir = os.path.dirname(os.path.abspath(pyname))
-    if plugin:
+    module_dir = os.path.dirname(os.path.abspath(requesting_file_path))
+    if is_plugin:
         from sideboard.lib import config
         plugin_name = os.path.basename(module_dir)
-        root_dir = os.path.join(config['plugins_dir'], plugin_name.replace('_', '-'))
+        root_dir = os.path.join(config['plugins_dir'], plugin_name)
```

Comment:
I wondered about this. We haven't settled on a standard here: underscores or hyphens in package names. It seems like you've settled on hyphens in your day job, which will probably end up driving this decision, but in our deployments we are still using underscores (for `attendee_tournaments` and `uber_analytics`). So we have something like this:
```
/opt/sideboard/plugins/attendee_tournaments/attendee_tournaments/__init__.py
/opt/sideboard/plugins/uber_analytics/uber_analytics/__init__.py
```

I opted to remove the call to `.replace()` as it fits MAGFests current deployment scheme, and I've seen many python packages that use underscores in the package name ([logging_unterpolation](https://pypi.python.org/pypi/logging_unterpolation) for example).

As a compromise, how about something like this?
```python
root_dir = os.path.join(config['plugins_dir'], plugin_name)
if '_' in plugin_name and not os.path.exists(root_dir):
    root_dir = os.path.join(config['plugins_dir'], plugin_name.replace('_', '-'))
```